Library/LMS.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library
+{
+	class LMS
+	{
+      
+
+        public static void Main(String[] args)
+		{
+			Operations bookOperations = new Operations();
+			Console.WriteLine("Enter the total number of Operations to be performed ");
+			String NoOperations = Console.ReadLine();
+			int noOperations = Convert.ToInt32(NoOperations);
+			Console.WriteLine("Total Operations to be performed: " + noOperations);
+
+			while (noOperations > 0)
+			{
+				Console.WriteLine("Enter 1 to add the books in the library");
+				Console.WriteLine("Enter 2 to remove the books in the library");
+				Console.WriteLine("Enter 3 to print the books available in the library");
+				Console.WriteLine("Enter 4 to search for a book in the library");
+				Console.WriteLine("Enter 5 to update a books in the library");
+				
+				String Input = Console.ReadLine();
+				int input = Convert.ToInt32(Input);
+				switch (input)
+				{
+					case 1:
+						
```

Comment:
this entire code for case 1 should come under single addBook() method.

Similarly for all other cases.

Reply:
I have made the changes in the code.